www/_posts/2023-05-22-cordova-cli-12.0.0.md

```diff
@@ -0,0 +1,103 @@
+---
+layout: post
+author:
+    name: Bryan Ellis
+title:  "Cordova CLI 12.0.0 & Internal Packages Released!"
+categories: news
+tags: news releases
+---
+
+We are happy to announce that we have just released a major update to our Cordova CLI!
+
+* [cordova@12.0.0](https://www.npmjs.com/package/cordova)
+
+In this CLI release, it also includes the latest internal libraries:
+
+* [cordova-lib@12.0.1](https://www.npmjs.com/package/cordova-lib)
+* [cordova-create@5.0.0](https://www.npmjs.com/package/cordova-create)
+* [cordova-common@5.0.0](https://www.npmjs.com/package/cordova-common)
+* [cordova-fetch@4.0.0](https://www.npmjs.com/package/cordova-fetch)
+* [cordova-serve@4.0.1](https://www.npmjs.com/package/cordova-serve)
+
+
+## Release Highlights
+
+This release drops Nodejs 14 support. The minimum supported version that Cordova requires is greater than or equal to 16.13.0.
+
+In all releases, we have updated all npm packages to the possible latest release that Cordova can support.
+
+We have removed the deprecated platform **OSX** & **Windows** from the platform listing and removed all platform pinnings so that future platform releases are available to use immediately.
+
+Please report any issues you find at [issues.cordova.io](http://issues.cordova.io/)!
+
+<!--more-->
+# Changes include:
+
```

Comment:
I had difficulty putting thoughts into words on this one... so feel free to update/change things around.... but hopefully the main point gets acrossed.

### START OF SNIPPET

```
**Breaking:**

## Platform Pinning

In previous versions of Cordova, the CLI used to pin platforms to a particular major version which will remained fixed for a given Cordova CLI major version. This meant that sometimes to get the latest version of platform, you'll need to explicitly specify the version. Starting with Cordova 12, we no longer pin platforms so that when adding new platforms, the `latest` version is always used. This is more consistent with how NPM handles packages. Based on common usage of Cordova CLI, we feel like this change will make it easier for developers getting onboard with Cordova.

If you rely on specific platform pinning, updates to your workflow/CI environments may be needed to explicitly pin to the desired versions.
```